src/rpc/blockchain.cpp

```diff
@@ -1047,13 +1047,26 @@ static RPCHelpMan pruneblockchain()
     };
 }
 
+CoinStatsHashType ParseHashType(const std::string& hash_type_input)
+{
+    if (hash_type_input == "hash_serialized_2") {
+        return CoinStatsHashType::HASH_SERIALIZED;
+    } else if (hash_type_input == "none") {
+        return CoinStatsHashType::NONE;
+    } else if (hash_type_input == "muhash") {
+        return CoinStatsHashType::MUHASH;
+    } else {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("%d is not a valid hash_type", hash_type_input));
```

Comment:
726c9504 I overlooked this in my first review
```suggestion

        throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("%s is not a valid hash_type", hash_type_input));
```

Reply:
Perhaps add a functional test for this.

Reply:
Oh, maybe it works anyway
```
./src/bitcoin-cli -signet gettxoutsetinfo a
error code: -8
error message:
a is not a valid hash_type
```

Reply:
Done